sound/soc/sof/sof-pci-dev.c

```diff
@@ -23,6 +23,10 @@ static char *fw_path;
 module_param(fw_path, charp, 0444);
 MODULE_PARM_DESC(fw_path, "alternate path for SOF firmware.");
 
+static char *fw_filename;
+module_param(fw_filename, charp, 0444);
+MODULE_PARM_DESC(fw_filename, "alternate path for SOF firmware name.");
```

Comment:
I'd rephrase this as "alternate filename for SOF firmware.".

Reply:
done